src/cli/cmd/login/mod.rs

```diff
@@ -126,24 +182,54 @@ impl LoginSubcommand {
                 .ok_or_else(|| color_eyre::eyre::eyre!("cache_addr had no host"))?,
         );
 
+        let mut succeeded = false;
+
+        if let Some(mut uds) = dnee_uds {
+            tracing::info!("trying to update netrc via DNEE");
+
+            let add_req = NetrcTokenAddRequest {
+                token: token.clone(),
+                netrc_lines: netrc_contents.clone(),
+            };
+            let add_req_json = serde_json::to_string(&add_req)?;
+            let request = http::request::Builder::new()
+                .uri("http://dnee.socket/enroll-netrc-token")
```

Comment:
```suggestion
                .uri("http://localhost/enroll-netrc-token")
```

the domain name doesn't really matter, so let's just use localhost

(this is what fly does for their socket-api thing too)

Reply:
I'm fine with localhost something, but can I make it `localhost-uds` at least? I realize it's still a misnomer, but :/

Reply:
okay, the reasoning on the next one convinced me, I'll just make them localhost